buildpipeline/pipelinejobs.groovy

```diff
@@ -17,15 +17,15 @@ def branch = GithubBranchName
 def linPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/linux.groovy')
 def linArm64Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/linux.arm64.groovy')
 def centos6Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/centos.6.groovy')
-def alpine36Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/alpine.3.6.groovy')
+def linmuslPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/linux-musl.groovy')
```

Comment:
(nit) I know it is used currently in this file, but the `lin` abbreviation just doesn't seem right to me.  I've never seen that before.

Reply:
It made more since when it was just "lin", "win", and "osx" it allowed the table to format easier as they were all the same length but now there are others so it make less sense. However I'm going to refrain from cleaning up this file because I want to keep this change scoped so I can port it to release/2.1.